test/notBoth.js

```diff
@@ -5,7 +5,20 @@ import * as RA from '../src/index';
 import eq from './shared/eq';
 
 
+const supportsFantasyLand = () => {
+  let result;
+  // Earlier versions of Ramda will throw.
+  try {
+    result = RA.notBoth(Just(true), Just(true));
+  } catch (e) {
+    return false;
+  }
+  return Just(false).equals(result);
+};
+
 describe('notBoth', function() {
+  const isFantasyLandSupported = supportsFantasyLand();
```

Comment:
If you don't want it to run on every test (which I agree doesn't make sense) just wrap it into [before](https://mochajs.org/#hooks) block to run it only once and semantically divides it.

```js
let  isFantasyLandSupported;

before(function () {
  isFantasyLandSupported = supportsFantasyLand();
});

Reply:
I tried this, but unfortunately the conditional around the suite is evaluated immediately, meaning the value of `isFantastyLandSupported` is `undefined`.

Reply:
Ahh..true, then disregard my comment.